src/components/dropdown/dropdown-iten/dropdown-item.css

```
@@ -0,0 +1,14 @@
+.item-container {
+    padding: 1rem;
+    display: flex;
+    flex-direction: row;
+    padding: 1rem;
+}
```

```suggestion
    gap: 15px;
}
```

Eu explico certinho ali embaixo